src/Workspaces/Core/Portable/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -99,23 +99,19 @@ public MemoryMappedInfo(string name, long offset, long size)
             /// </summary>
             public Stream CreateReadableStream()
             {
-                // CreateViewAccessor is not guaranteed to be thread-safe
```

Comment:
📝 The implementation just calls [`MapViewOfFile`](https://docs.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-mapviewoffile), so we need to verify that the OS method is thread safe for the same handle.

Reply:
Based on the remarks here, I think this is a safe change:
https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createfilemappinga#remarks

Reply:
@jaredpar @stephentoub How would we get confirmation of this?

Reply:
Internal feedback is this call is already going to be synchronized by the kernel, and therefore the second layer of synchronization within the application is unnecessary.